src/test/java/com/wikia/webdriver/elements/fandom/components/LoadMore.java

```diff
@@ -0,0 +1,35 @@
+package com.wikia.webdriver.elements.fandom.components;
+
+import com.gargoylesoftware.htmlunit.ElementNotFoundException;
+import org.openqa.selenium.By;
```

Comment:
If you want to get rid of sonar comments, just log info passing the exception object before returning false.